rasa/shared/core/training_data/story_reader/story_reader.py

```diff
@@ -62,18 +56,6 @@ def read_from_file(
         """
         raise NotImplementedError
 
-    @staticmethod
-    def is_test_stories_file(filename: Text) -> bool:
```

Comment:
I'm slightly confused why this method was removed 🤔

Reply:
No reason for it to exist because it's just a static method that just proxies its arguments to another function.

Reply:
Ah I see, you're referring to its implementation in `YAMLStoryReader`.